### PR TITLE
Print information about types of errors.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: test/libsolidity/SolidityNameAndTypeResolution.cpp [hunk 1]
```diff
@@ -104,7 +104,14 @@ parseAnalyseAndReturnError(string const& _source, bool _reportWarnings = false,
 			if (!StaticAnalyzer(errorReporter).analyze(*sourceUnit))
 				success = false;
 		if (errorReporter.errors().size() > 1 && !_allowMultipleErrors)
-			BOOST_FAIL("Multiple errors found");
+		{
+			string message = "Multiple errors found: ";
+			for (auto const& e: errorReporter.errors())
+				if (string const* description = boost::get_error_info<errinfo_comment>(*e))
+					message += *description + ", ";
```
Comment (axic) [inline test/libsolidity/SolidityNameAndTypeResolution.cpp]: comma or new line to separate them?

**Comment by axic** [comment]
I think this could be added to most tests which require a single error. Though this really calls for a helper to pull that code into a single place.

**Comment by chriseth** [comment]
@axic please request changes and do a proper review :-)

**Comment by chriseth** [comment]
I mean either say "it's fine, can be merged" or "this and that has to be changed"

**Review by axic — CHANGES_REQUESTED** [review]
Please make this a helper and it to all tests.

**Review by axic — APPROVED** [review]

**Pull request merged**